### PR TITLE
make port classification configurable

### PR DESCRIPTION
**Pull request by noamblitz**
### Changes
It is now configurable with config a config ooi on network

### Issue link
Closes https://github.com/minvws/nl-kat-coordination/issues/975

To qa you can create this config ooi and create a port that is a database port by default. Now it should be an uncommon port.

image

Will add unit test

### Code Checklist
- [x] All the commits in this PR are properly PGP-signed and verified;
- [x] This PR only contains functionality relevant to the issue; tickets have been created for newly discovered issues.
- [x] I have written unit tests for the changes or fixes I made.
- [x] For any non-trivial functionality, I have added integration and/or end-to-end tests.
- [x] I have performed a self-review of my code and refactored it to the best of my abilities.

### Communication
- [ ] I have informed others of any required `.env` changes files if required and changed the `.env-dist` accordingly.
- [ ] I have made corresponding changes to the documentation, if necessary.

---
## Checklist for code reviewers:
Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_code.md) into your comment.

---
## Checklist for QA:
Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_qa.md) into your comment.

### CODE REVIEW

**Review by ammar92 — DISMISSED** [review]

File: docs/source/manual/usermanual.rst [hunk 1]
```diff
@@ -435,3 +435,15 @@ will be aggregated into one finding, instead of one separate finding per port.
         "bit-id": "port-classification-ip",
         "config": {"aggregate_findings": "True"}
     }
+
+Also you can configure which open ports should create findings and which ports should not. This is done by settings
+common_tcp_ports, common_udp_ports, sa_tcp_ports and/or db_tcp_ports. As an example:
```
Comment (Darwinkel) [inline docs/source/manual/usermanual.rst:440]: What does `sa_tcp_ports` mean? What is the difference between the four setting lists? 

Also, from this wording it's unclear whether defining these ports here means that they will trigger findings or not. Maybe rephrase to something like: _"Any port defined in `common_tcp_ports` will no longer trigger an uncommon open port finding."_

Comment (noamblitz) [reply docs/source/manual/usermanual.rst:440]: Makes sense! Will rephrase!

**Comment by Darwinkel** [comment]
## Checklist for QA:
- [x] I have checked out this branch, and successfully ran a fresh `make reset`.
- [x] I confirmed that there are no unintended functional regressions in this branch:
    - [x] I have managed to pass the onboarding flow
    - [x] Objects and Findings are created properly
    - [x] Tasks are created and completed properly
- [x] I confirmed that the PR's advertised `feature` or `hotfix` works as intended.

### What works:
* Works in conjunction with #1407 

### Bug or feature?:
* I had some notes about the documentation, but those have been processed.

**Comment by noamblitz** [comment]
Will work on accepting empty string as config and extending docs as discussed

**Review by dekkers — APPROVED** [review]
